test/integration/webhook/v1alpha1/workload_test.go

```diff
@@ -70,3 +70,76 @@ var _ = ginkgo.Describe("Workload defaulting webhook", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("Workload validating webhook", func() {
+	ginkgo.Context("When creating a Workload", func() {
+		ginkgo.It("Should validate spec.podSets.count", func() {
+			ginkgo.By("Creating a new Workload")
+			workload := testing.MakeWorkload("workload2", ns.Name).Obj()
+			workload.Spec.PodSets = []v1alpha1.PodSet{
+				{
+					Count: -1,
+					Spec: corev1.PodSpec{
+						Containers: []corev1.Container{
+							{
+								Name:  "test",
+								Image: "fake-image",
+							},
+						},
+					},
+				},
+			}
+			err := k8sClient.Create(ctx, workload)
+			gomega.Expect(err).Should(gomega.HaveOccurred())
+			gomega.Expect(err.Error()).Should(gomega.ContainSubstring("must be greater than 0"))
+		})
+
+		ginkgo.It("Should validate spec.priorityClassName", func() {
+			ginkgo.By("Creating a new Workload")
+			workload := testing.MakeWorkload("workload3", ns.Name).Obj()
+			workload.Spec.PriorityClassName = "invalid_class"
```

Comment:
I am not sure if I understand what you mean correctly. Did you suggest we should create a `PriorityClass` object here rather than setting the `.spec.PriorityClassName` field of the `Workload`?

Reply:
Ah sorry for the mistake, I paste the wrong link, here's the right one:
https://github.com/kubernetes-sigs/kueue/blob/99683618876a004f3332fb99cae1e3a9bf6f2124/pkg/util/testing/wrappers.go#L189-L192
So we can simplify like this:
```
workload := testing.MakeWorkload("workload3", ns.Name).PriorityClass("invalid_class").Obj()
```